index.js

```diff
@@ -232,10 +233,11 @@ module.exports = (api, options) => {
             api.resolve(mainProcessFile),
             api.resolve(`${outputDir}/bundled/background.js`)
           )
-          buildApp()
+          buildApp(args.skipElectronBuild)
         }
       }
-      function buildApp () {
+      function buildApp (skip) {
+        if (skip) return
```

Comment:
May be ideal to also log that the electron-builder step is being skipped. Similar to how its done [here](https://github.com/nklayman/vue-cli-plugin-electron-builder/pull/1237/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L101)

```
        console.log('Not building app as --skipElectronBuild was passed')
```

Reply:
Good catch!  Updated